python3-hello/README.md

```diff
@@ -0,0 +1,407 @@
+Make sure you installed the [requirements](../README.md#requirements).
+
+## Quick Setup (aka TLDR)
+
+For a quick setup, run the commands below.
+Note that you still need to install the [requirements](../README.md#requirements).
+
+To build and run the application for `x86_64`, use the commands below:
+
+```console
+test -d ../repos/unikraft || git clone https://github.com/unikraft/unikraft ....//repos/unikraft
+test -d ../repos/libs/musl || git clone https://github.com/unikraft/lib-musl ../repos/libs/musl
+test -d ../repos/libs/lwip || git clone https://github.com/unikraft/lib-lwip ../repos/libs/lwip
+test -d ../repos/libs/python3 || git clone https://github.com/unikraft/lib-python3 ../repos/libs/python3
+test -d ../repos/libs/compiler-rt || git clone https://github.com/unikraft/lib-compiler-rt ../repos/libs/compiler-rt
+make distclean
+> /tmp/defconfig echo 'CONFIG_PLAT_KVM=y
+CONFIG_KVM_VMM_QEMU=y
+CONFIG_ARCH_X86_64=y
+CONFIG_LIBPYTHON3=y
+CONFIG_LIBPYTHON3_MAIN_FUNCTION=y
+CONFIG_LIBVFSCORE=y
+CONFIG_LIBVFSCORE_AUTOMOUNT_UP=y
+CONFIG_LIBRAMFS=y
+CONFIG_LIBUKCPIO=y
+CONFIG_STACK_SIZE_PAGE_ORDER=10
+CONFIG_LIBPOSIX_ENVIRON=y
+CONFIG_LIBPOSIX_ENVIRON_LIBPARAM=y'
+UK_DEFCONFIG=/tmp/defconfig make defconfig
+make -j $(nproc)
+test -d ./rootfs/ || docker build -o ./rootfs -f Dockerfile .
+test -f initrd.cpio || ../repos/unikraft/support/scripts/mkcpio initrd.cpio ./rootfs/
+qemu-system-x86_64 \
+    -nographic \
+    -m 256 \
+    -cpu max \
+    -append "python3-hello_qemu-x86_64 vfs.fstab=[ \"initrd0:/:extract::ramfs=1:\" ] -- /app/hello.py" \
+    -kernel out/python3-hello_qemu-x86_64 \
+    -initrd ./initrd.cpio
+```
+
+This will configure, build and run Python3 Hello on Unikraft.
+You can see how to test it in the ["Test" section](#test).
+To close the virtual machine, see the instructions in the ["Close QEMU" section](#close-qemu).
+
+To do the same for `AArch64`, run the commands below:
+
+```console
+test -d ../repos/unikraft || git clone https://github.com/unikraft/unikraft ....//repos/unikraft
+test -d ../repos/libs/musl || git clone https://github.com/unikraft/lib-musl ../repos/libs/musl
+test -d ../repos/libs/lwip || git clone https://github.com/unikraft/lib-lwip ../repos/libs/lwip
+test -d ../repos/libs/python3 || git clone https://github.com/unikraft/lib-python3 ../repos/libs/python3
+test -d ../repos/libs/compiler-rt || git clone https://github.com/unikraft/lib-compiler-rt ../repos/libs/compiler-rt
+make distclean
+> /tmp/defconfig echo 'CONFIG_PLAT_KVM=y
+CONFIG_KVM_VMM_QEMU=y
+CONFIG_ARCH_ARM_64=y
+CONFIG_LIBPYTHON3=y
+CONFIG_LIBPYTHON3_MAIN_FUNCTION=y
+CONFIG_LIBVFSCORE=y
+CONFIG_LIBVFSCORE_AUTOMOUNT_UP=y
+CONFIG_LIBRAMFS=y
+CONFIG_LIBUKCPIO=y
+CONFIG_STACK_SIZE_PAGE_ORDER=10
+CONFIG_LIBPOSIX_ENVIRON=y
+CONFIG_LIBPOSIX_ENVIRON_LIBPARAM=y
+CONFIG_ARM64_ERRATUM_858921=n
+CONFIG_ARM64_ERRATUM_835769=n
+CONFIG_ARM64_ERRATUM_843419=n'
+UK_DEFCONFIG=/tmp/defconfig make defconfig
```

Comment:
This does not work for me. `CONFIG_LIBPYTHON3_PYTHONPATH=` should be set to `/usr/local/lib/python3.10:/usr/local/lib/python3.10/site-packages`, otherwise it defaults to `/lib/python3.10`, which does not exist in the rootfs.
```suggestion
CONFIG_LIBPYTHON3_PYTHONPATH="/usr/local/lib/python3.10:/usr/local/lib/python3.10/site-packages"'
UK_DEFCONFIG=/tmp/defconfig make defconfig
```

Reply:
I missed adding the `env.vars` variable to the command line.